icacls_files/credentials.rb

```diff
@@ -0,0 +1,8 @@
+require 'yaml'
+# class for working with credentials
+module Credential
+  def get_credentials
+    credentials = YAML.load_file('icacls_files/credentials.yml')
+    credentials
```

Comment:
You can remove 'credentials' and 'credentials = '

Reply:
exactly

Reply:
when you're available please check some changes